### PR TITLE
hw-mgmt: global wp: Extend global write protection script

### PR DESCRIPTION
**Pull request by sholeksandr**
Add support P4697 system with separate {}_wp bits per ASIC

Signed-off-by: Oleksandr Shamray <oleksandrs@nvidia.com>

### CODE REVIEW

File: usr/usr/bin/hw-management-global-wp.sh [hunk 1]
```diff
@@ -103,6 +103,53 @@ do_global_wp_release_restore()
 	fi
 }
 
+
```
Comment (ciju-nvidia) [inline usr/usr/bin/hw-management-global-wp.sh:106]: May be a description would be good here

File: usr/usr/bin/hw-management-global-wp.sh [hunk 2]
```diff
@@ -138,7 +185,13 @@ release|restore)
 		exit 1
 	fi
 	echo "$global_wp_pid" > /var/run/hw-management-global-wp.pid
-	do_global_wp_release_restore "$action" "$2" "$3" "$4" "$5" "$6" "$7"
+	# systems without global WP but with ASIC WP
+	sku=$(< /sys/devices/virtual/dmi/id/product_sku)
```
Comment (ciju-nvidia) [inline usr/usr/bin/hw-management-global-wp.sh:189]: Does it make sense to check the if 'sku' is valid?

**Review by vadimp-nvidia — APPROVED** [review]

**Comment by vadimp-nvidia** [comment]
RERUN

File: usr/usr/bin/hw-management-global-wp.sh (hunk 2 above)
Comment (sholeksandr) [reply usr/usr/bin/hw-management-global-wp.sh:189]: No. 
1. We running do_asic_wp_release_restore onlu if SKU == HI142. In all other cases we running default do_global_wp_release_restore.
2. usr/usr/bin/hw-management-global-wp.sh can by called only on systems with SKU present.

**Review by ciju-nvidia — DISMISSED** [review]

**Pull request merged**